### PR TITLE
Oadp1306br

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP 1.1.6

OCP 4.11+

Resolves - https://issues.redhat.com/browse/OADP-1306

Deploy preview - https://64083--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs#oadp-about-backup-snapshot-locations_installing-oadp-ocs

The updates appear in the following sections:
1. Secrets

2. Creating an Object Bucket Claim for disaster recovery on OpenShift Data Foundation.

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc [hunk 1]
```diff
@@ -20,6 +20,7 @@ To install the OADP Operator in a restricted network environment, you must first
 include::modules/oadp-installing-operator.adoc[leveloffset=+1]
 include::modules/migration-configuring-aws-s3.adoc[leveloffset=+1]
 include::modules/oadp-about-backup-snapshot-locations-secrets.adoc[leveloffset=+1]
+
```
Comment (kaovilai) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc:23]: Is this line necessary?

**Review by kaovilai — APPROVED** [review]

**Comment by CarmiWisemon** [comment]
@GroceryBoyJr 
The release date for this PR is today.
Can I send it for merge review?

**Pull request merged**